doc/man3/OSSL_ALGORITHM.pod

```diff
@@ -102,6 +102,15 @@ A string with a short human-readable description of the algorithm.
 
 =head2 On the subject of algorithm names
 
+All the names necessary for using an algorithm should be provided by the
+OSSL_ALGORITHM structure. Symbolic names, ahort and long, are necessary for
+looking up by name. OID is necessary for usage in ASN.1 structures etc. In
+case when the names are well-known (e.g. present in OID database of OpenSSL),
+the developers aren't obliged to provide anythings besides short name.
```

Comment:
This basically repeats a lot of the text that's already here, does it not?

I was under the impression that you mostly wanted to make the last paragraph (quoted here) stronger, did I misunderstand?

> It's recommended to make sure that the corresponding **OSSL_ALGORITHM**'s *algorithm_names* include known names as well as the OID itself in canonical decimal text form.  That should cover all scenarios.